app/views/articles/_sidebar_campaign.html.erb

```diff
@@ -1,16 +1,19 @@
 <div class="widget">
+  <% if SiteConfig.campaign_sidebar_image %>
+    <img src="<%= SiteConfig.campaign_sidebar_image %>" />
+  <% end %>
   <header>
     <h4>
-      <% SiteConfig.campaign_featured_tags.each do |t| %>
-        <%= link_to("##{t}", "/t/#{t}") %>
-      <% end %>
+      <%= link_to "Stories (#{@campaign_articles_count})", "/t/#{SiteConfig.campaign_featured_tags.first}" %>
     </h4>
   </header>
   <div class="widget-body">
     <div class="widget-link-list">
       <% @latest_campaign_articles.each do |plucked_article| %>
-        <%= render "articles/widget_list_item", plucked_article: plucked_article, show_comment_count: false %>
+        <%= render "articles/widget_list_item", plucked_article: plucked_article, show_comment_count: true %>
       <% end %>
     </div>
+    <%= link_to "Share your story", "/new/#{SiteConfig.campaign_featured_tags.first}", class: "write" %>
```

Comment:
I decided not to make these links configurable (besides the tag) though we may need smth different for other campaigns. Because if we make them configurable right now, we may still need different type of configuration later.